cypress/integration/ui/sign.test.ts

```diff
@@ -0,0 +1,21 @@
+/// <reference types="cypress" />
+/// <reference types="@testing-library/cypress"/>
+import { paymentXdr, sourceAccount, paymentRecipient } from '../../fixtures/sign.json';
+
+const url = Cypress.env('HOST');
+describe('sign user history', () => {
```

Comment:
what history 🤔 ?

Reply:
I meant story, sorry. Anyway it's badly worded, I'm gonna give it a better name.